app/src/main/java/io/mrarm/irc/IgnoreListAdapter.java

```diff
@@ -94,6 +100,10 @@ private void startEdit() {
 
         public void bind(ServerConfigData.IgnoreEntry entry) {
             ColoredTextBuilder builder = new ColoredTextBuilder();
+            if (entry.isBad) {
+                builder.append("@bad:", new ForegroundColorSpan(Color.RED), new StyleSpan(Typeface.BOLD));
```

Comment:
This should probably use an android string for the prefix? Also it should probably read `"Bad: "` not `"@bad:"`